tests/clam.py

```diff
@@ -30,6 +31,14 @@ class Species(proto.Enum):
     GIGAS = 3
 
 
+class Color(proto.Enum):
+    COLOR_UNKNOWN = 0
```

Comment:
```suggestion
    UNKNOWN = 0
```

Reply:
`COLOR_UNKNOWN` follows protobuf best practices for the default value

`In proto3, the first value defined in an enum definition must have the value zero and should have the name ENUM_TYPE_NAME_UNSPECIFIED or ENUM_TYPE_NAME_UNKNOWN`

https://protobuf.dev/programming-guides/proto3/#enum-default

Reply:
Sounds good!